### PR TITLE
eight_ball command

### PR DESCRIPTION
**Pull request by yannicfreson**
Command that randomly gives one of 20 possible answer like a Magic Eight Ball does. Answers are fetched from an API. Link to the API on GitHub: https://github.com/yannicfreson/EightBall-API

### CODE REVIEW

**Review by bobvdvleuten — CHANGES_REQUESTED** [review]
Would put that API url in constants.py

**Comment by bobvdvleuten** [comment]
Seperately from my requested change, a remark on that API url itself, is that subdomain going to reliably remain permanent like that?

**Comment by yannicfreson** [comment]
Should I also put the url for the dadjoke api used in that command in the constants.py then? Because I didn't get that comment there.
As for the url, unless I change it myself it shouldn't change.

**Comment by bobvdvleuten** [comment]
@yannicfreson I already made a patch PR for that here: https://github.com/nerdland-unofficial-fans/nerdlandbot/pull/138

**Comment by yannicfreson** [comment]
Alright nice! I'll make the same change here then :)

File: nerdlandbot/translations/Translations.csv [hunk 1]
```diff
@@ -135,6 +135,7 @@
...
 "open_source_message","Hello, I'm open source. You can help me become a better me by opening issues or writing software that I need to solve those issues. If you don't know where to start, open an issue by clicking on this link: https://github.com/nerdland-unofficial-fans/nerdlandbot/issues/new/choose.","Hallo, deze bot is open source. Je kan helpen om me beter te maken door issues aan te maken of software te schrijven die ik nodig heb om te functioneren. Als je niet weet waar je kan beginnen, kan je een issue starten door te klikken op deze link: https://github.com/nerdland-unofficial-fans/nerdlandbot/issues/new/choose."
+"eight_ball_help","Replies a random answer from magic eight ball","Antwoordt met een willekeurig antwoord van magic eight ball"
```
Comment (bobvdvleuten) [inline nerdlandbot/translations/Translations.csv:138]: van "de" magic eight ball? (nitpicking, not blocking my approval.)

**Review by bobvdvleuten — APPROVED** [review]

**Review by Eva774 — APPROVED** [review]

**Pull request merged**